<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadowWindow</class>
 <widget class="QMainWindow" name="ShadowWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShadowWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_between_Seacrh_Computer">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>-20</y>
      <width>20</width>
      <height>621</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_between_Computer_User">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>-10</y>
      <width>20</width>
      <height>621</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>461</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>MyComputer</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>401</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>CPU</string>
     </property>
     <widget class="QLabel" name="label_IMAGE_cpu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>image_cpu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_cpu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_gpu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacture_set_cpu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>51</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_IMAGE_myComputer_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Image </string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>401</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>GPU</string>
     </property>
     <widget class="QLabel" name="label_IMAGE_gpu">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>image_gpu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_set_gpu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>45</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>401</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Storage</string>
     </property>
     <widget class="QLabel" name="label_image_storage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>image_strg</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_set_storage">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>46</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>401</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Memorychip</string>
     </property>
     <widget class="QLabel" name="label_IMAGE_memorychip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>image_mc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_mamorychip">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_set_memorychip">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>401</width>
       <height>90</height>
      </rect>
     </property>
     <property name="title">
      <string>Baseboard</string>
     </property>
     <widget class="QLabel" name="label_IMAGE_baseboard">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>image_bas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_baseboard">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_manufacturer_set_baseboard">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_CPU">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>31</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_GPU">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>31</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_STORAGE">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>240</y>
       <width>31</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_MEMORYCHIP">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>330</y>
       <width>31</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_BASEBOARD">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>430</y>
       <width>31</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit_Seacrh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Insert and push right button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Search">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Order">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No order</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stars</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manufacturer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>isSale</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Older User</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Order">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Order:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Count">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>count:</string>
    </property>
   </widget>
   <widget class="Line" name="line_Search">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Count_set">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line_Down">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>530</y>
      <width>1291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>863</x>
      <y>11</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>USERNAME:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_set">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MyDelivery">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>My Delivery &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MyBankcard">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>230</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>My Bankcard &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AboutMe">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>270</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>About me &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Chats">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>310</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Chats &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Telegram">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>348</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Telegram &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_HelpMe">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>500</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Help me &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MyReview">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>420</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>My reviews &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Manual">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>460</y>
      <width>401</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual &gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_review">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Review:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_reviwcount_set">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IMAGE_review">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>40</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_IMAGE_Seacrh">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>551</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IMAGE_myComputer">
    <property name="geometry">
     <rect>
      <x>566</x>
      <y>550</y>
      <width>79</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>My computer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IMAGE_profile">
    <property name="geometry">
     <rect>
      <x>1057</x>
      <y>547</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MyReview_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>382</y>
      <width>401</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>add review youself (test)  &gt;&gt;&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionMy_Review"/>
    <addaction name="actionChat"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QMenu" name="menuDelivery">
     <property name="title">
      <string>Delivery</string>
     </property>
     <addaction name="actionadd"/>
    </widget>
    <widget class="QMenu" name="menuAdress">
     <property name="title">
      <string>Adress</string>
     </property>
     <addaction name="actionreset_set_adress"/>
    </widget>
    <addaction name="menuDelivery"/>
    <addaction name="menuAdress"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionPush"/>
    <addaction name="actionPull"/>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuData"/>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPush">
   <property name="text">
    <string>Push</string>
   </property>
  </action>
  <action name="actionPull">
   <property name="text">
    <string>Pull</string>
   </property>
  </action>
  <action name="actionMy_Review">
   <property name="text">
    <string>My Review</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="text">
    <string>Chat</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionreset_set_adress">
   <property name="text">
    <string>reset|set adress</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
